functions/command/add.go

```diff
@@ -30,7 +32,37 @@ func (c *AddCommand) Execute(ctx context.Context, cl *clients.Clients) (*discord
 	case "activity":
 		typ = activity.ACTIVITY
 	case "game":
-		return utils.NewWebhookEdit("🚧 Games not implemented yet"), nil
+		typ = activity.GAME
+
+		ctx, cancel := context.WithTimeout(context.Background(), time.Duration(time.Millisecond*500))
+		defer cancel()
+
+		detail, err := cl.Rawg().GetGame(ctx, c.Name)
+
+		if err != nil {
+			msg := clients.RawgErrorMsg(err)
+			return utils.NewWebhookEdit(msg), err
```

Comment:
Should return a nil error when the game is not found. That is a user error and not an error on our end